### PR TITLE
Add Add&SetCommands

### PR DESCRIPTION
**Pull request by simongraband**
Signed-off-by: Simon Graband <simon.graband@tum.de>

### CODE REVIEW

File: server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java [hunk 1]
```diff
@@ -97,16 +99,78 @@ private EPackage getEPackage(EcoreModelState modelState) {
...
+	
+	public boolean setAttributeType(EcoreModelState modelState, EAttribute eAttribute, EAttribute type) {
+		return this.set(modelState, eAttribute, EcorePackage.eINSTANCE.getEAttribute_EAttributeType(), type);
+	}
+	
+	public boolean setEdgeName(EcoreModelState modelState, EReference eReference, String name) {
+		return this.set(modelState, eReference, EcorePackage.eINSTANCE.getENamedElement_Name(), name);
+	}
+	
+	public boolean setLowerMultiplicity(EcoreModelState modelState, EReference eReference, int multiplicity) {
```
Comment (CamilleLetavernier) [inline server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java:153]: This method can be generalized to accept any EStructuralFeature (i.e. either an EAttribute or an EReference). While the diagram doesn't show the EAttribute multiplicity yet, we'll probably add it in the future (Via the EAttribute text parser, similar to the EAttribute Type)

File: server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java [hunk 2]
```diff
@@ -97,16 +99,78 @@ private EPackage getEPackage(EcoreModelState modelState) {
...
+		return this.set(modelState, eReference, EcorePackage.eINSTANCE.getETypedElement_LowerBound(), multiplicity);
+	}
+	
+	public boolean setUpperMultiplicity(EcoreModelState modelState, EReference eReference, int multiplicity) {
```
Comment (CamilleLetavernier) [inline server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java:157]: Same thing here

File: server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java [hunk 3]
```diff
@@ -97,16 +99,78 @@ private EPackage getEPackage(EcoreModelState modelState) {
...
+		return this.set(modelState, eReference, EcorePackage.eINSTANCE.getETypedElement_UpperBound(), multiplicity);
 	}
 
-	private boolean add(EcoreModelState modelState, EObject owner, EReference feature, EObject element) {
+	private boolean add(EcoreModelState modelState, EObject owner, EReference feature, Object addObject) {
```
Comment (CamilleLetavernier) [inline server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java:161]: Do we actually have a use-case where we add an element that is not an EObject? (In general, it's possible when editing the value of N-ary EAttributes, but I don't think Ecore has any)

**Review by CamilleLetavernier — APPROVED** [review]
Looks good to me! Just some minor API comments

File: server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java (hunk 3 above)
Comment (simongraband) [reply server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java]: Actually not i changed this for Strings before having the Set-Command and forgot to change it back. I reverted it.

**Review by CamilleLetavernier — APPROVED** [review]
Thanks!

**Pull request merged**